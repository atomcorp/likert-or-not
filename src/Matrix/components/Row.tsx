import * as React from 'react';

interface RowProps {
  children?: React.ReactNode;
  className?: string;
}

const Row = (props: RowProps) => (
  <tr className={props.className}>{props.children}</tr>
);

export default Row;
