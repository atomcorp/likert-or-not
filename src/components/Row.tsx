import * as React from 'react';
import LikertOrNot from '../index.d';

const Row = (props: LikertOrNot.RowProps) => (
  <tr className={props.className}>{props.children}</tr>
);

export default Row;
