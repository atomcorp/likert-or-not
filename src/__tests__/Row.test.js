import React from 'react';
import Row from '../components/Row';
import renderer from 'react-test-renderer';

test('Title prints text in caption', () => {
  const component = renderer.create(
    <Row className={'exampleClass'}>
      <div>A node</div>
    </Row>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
