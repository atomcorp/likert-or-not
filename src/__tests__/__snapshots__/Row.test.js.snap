// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title prints text in caption 1`] = `
<tr
  className="exampleClass"
>
  <div>
    A node
  </div>
</tr>
`;
